stl/inc/regex

```diff
@@ -3491,8 +3501,8 @@ bool _Matcher3<_BidIt, _Elem, _RxTraits, _It, _Alloc>::_Find_first_inner_capture
 
         case _N_rep:
             {
-                _Node_rep* _Inner_rep              = static_cast<_Node_rep*>(_Nx);
-                _Loop_vals_v2_t* _Inner_loop_state = &_Loop_vals[_Inner_rep->_Loop_number];
+                _Node_rep* _Inner_rep   = static_cast<_Node_rep*>(_Nx);
+                auto* _Inner_loop_state = &_Loop_vals[_Inner_rep->_Loop_number];

```

Comment:
Pre-existing: `_Inner_rep` is fine but since I'm going to make `_Inner_loop_state` a `const auto`, I'll change `_Inner_rep` too so they line up nicely.